### PR TITLE
Add instructions for manually creating a CRL

### PR DESCRIPTION
**Pull request by tashian**

### CODE REVIEW

File: src/pages/docs/step-ca/certificate-authority-server-production.mdx [hunk 1]
```diff
@@ -162,6 +163,88 @@ So do some certificates.
...
+  ```json
+  {
+          "subject": {{ toJson .Subject }},
+          "keyUsage": ["certSign", "crlSign"],
+          "basicConstraints": {
+                  "isCA": true,
+                  "maxPathLen": 0
+          },
+          "crlDistributionPoints":
+                  ["http://crl.example.com/crl/ca.crl.pem"]
```
Comment (maraino) [inline src/pages/docs/step-ca/certificate-authority-server-production.mdx]: The `DER` format is more common here, just name the file `ca.crl`

File: src/pages/docs/step-ca/certificate-authority-server-production.mdx [hunk 2]
```diff
@@ -162,6 +163,88 @@ So do some certificates.
...
+you can independently manage your own CRL if you like.
+
+#### Create an intermediate CA with a CRL distribution endpoint
+
+Let's make it possible to revoke your intermediate CA down the road if necessary.
+This setup is more complex than the default `step-ca` PKI, but it offers an insurance policy for a compromised intermediate CA.
+
+1. Create an intermediate CA that includes a CRL endpoint. Save the following template to `intermediate.tpl`:
+
+  ```json
```
Comment (maraino) [inline src/pages/docs/step-ca/certificate-authority-server-production.mdx:188]: GitHub prints this better if you use yaml here.

File: src/pages/docs/step-ca/certificate-authority-server-production.mdx [hunk 3]
```diff
@@ -162,6 +163,88 @@ So do some certificates.
...
+  default_crl_days = 30
+  database = index.txt
+  default_md = sha256
+  EOF
+  touch index.txt
+  openssl ca -config openssl.conf \
+      -gencrl \
+      -keyfile root_ca_key \
+      -cert root_ca.crt \
+      -out intermediate.crl
```
Comment (maraino) [inline src/pages/docs/step-ca/certificate-authority-server-production.mdx]: You named this before ca.crl

**Review by maraino — CHANGES_REQUESTED** [review]
Just a few details. 

At some point we should implement something like `step crl create`, it's not hard to do.

**Review by maraino — APPROVED** [review]
LGTM, but you haven't explained how to use the index.txt

**Comment by tashian** [comment]
Good point @maraino, I've just added a small bit about how to revoke a certificate

**Review by maraino — APPROVED** [review]
lgtm

**Pull request merged**